templates/python/pyproject.mustache

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "{{{pythonPackageName}}}"
-version = "{{{packageVersion}}}"
+version = "v0.0.1a"
```

Comment:
Do we need to pin a specific version in the generator?

Reply:
Not necessarily here.
But it's pinned anyway. The `packageVersion` variable is from the generator. If you don't set it in the CLI call it defaults to `1.0.0`.
Creating a new package will have a version change (obviously. Because there was no version before that), which will trigger a release. 
I don't think the initial release should have version `1.0.0`.
You probably have to manually adjust this in the created PR, but setting it to the lowest possible version is the safest option in case you forget to adjust.

Reply:
Ah for the creation of new packages it makes sense definitively, nice 👍

Reply:
But what about package updates? Won't this try to set the version back to `v0.0.1a`

Reply:
No. This file won't be adjusted for existing packages.
Specifically to prevent version changes.